nexuse2e-webapp/src/main/frontend/src/app/services/data.service.ts

```diff
@@ -58,25 +58,23 @@ export class DataService {
   private static buildFilterParams(activeFilters: ActiveFilterList) {
     let httpParams = new HttpParams();
     for (const fieldName in activeFilters) {
-      if (activeFilters.hasOwnProperty(fieldName)) {
-        const value = activeFilters[fieldName];
-        if (value) {
-          if (typeof value === "string") {
-            httpParams = httpParams.append(fieldName, value);
-          } else {
-            // type is DateRange
-            if (value.startDate) {
-              httpParams = httpParams.append(
-                "startDate",
-                value.startDate.toISOString()
-              );
-            }
-            if (value.endDate) {
-              httpParams = httpParams.append(
-                "endDate",
-                value.endDate.toISOString()
-              );
-            }
+      if (activeFilters.hasOwnProperty(fieldName)) {const value = activeFilters[fieldName];
```

Comment:
This indentation looks odd. We go one level deeper with the if, but the next if is on the same line. And I see no difference in the code, only this changes that makes it harder to read